server/service/src/processors/central_records/mod.rs

```diff
@@ -27,21 +26,39 @@ pub(crate) enum ProcessCentralRecordsError {
 
 const CHANGELOG_BATCH_SIZE: u32 = 20;
 
+#[derive(Clone)]
+pub enum CentralRecordProcessorType {
```

Comment:
Type is used because processor will be triggered in generic way, and need to specify which records to process